lib/cn/csched_sp3.c

```diff
@@ -323,7 +323,7 @@ scale2dbl(u64 samp)
 static inline uint
 samp_est(struct cn_samp_stats *s, uint scale)
 {
-    return scale * safe_div(s->i_alen + s->l_alen, s->l_good);
+    return scale * safe_div((s->i_alen / 4) + s->l_alen, s->l_good);
```

Comment:
what is the 4 here?

Reply:
That just reduces the impact of the root node on space amp by 4x.  Alex is going to get rid of this altogether, but this gets us closer to how we expect to be operating in the near future.  Basically it reduced space-amp thrashing which can trigger unnecessary gc.

Reply:
Sounds good